spring/boot3-autoconfigure/src/main/java/com/linecorp/armeria/spring/AbstractArmeriaAutoConfiguration.java

```diff
@@ -69,16 +69,16 @@ public Server armeriaServer(
             ArmeriaSettings armeriaSettings,
             InternalServices internalService,
             Optional<MeterRegistry> meterRegistry,
```

Comment:
Question: Can we also use `ObjectProvider` for this type?

Reply:
Yes, we can. But it may require something like `meterIdPrefixFunction.getIfAvailable() ? meterIdPrefixFunction.getIfAvailable() : MeterIdPrefixFunction.ofDefault("armeria.server")`, seems not so concise.

Reply:
Yeah, let's keep it as is. Thanks!